create-derived/index.js

```diff
@@ -1,20 +1,43 @@
-import { createStore } from '../create-store/index.js'
 import { getValue } from '../get-value/index.js'
+import { createStore } from '../create-store/index.js'
+
+const collectWritable = deps => [
+  ...new Set(
+    deps.reduce(
+      (acc, dep) => (dep.deps ? [...acc, ...dep.deps] : [...acc, dep]),
+      []
+    )
+  )
+]
 
 export function createDerived(stores, cb) {
   if (!Array.isArray(stores)) stores = [stores]
+  let deps = collectWritable(stores)
+
+  let run = () => {
+    let values = stores.map(store =>
+      store.wrapper ? store.wrapper() : getValue(store)
```

Comment:
Who can set `store.wrapper`?

Reply:
Resolved. I succeed to reduce the additional size from 54 to 35 B

Reply:
😍